pkg/capabilities/consensus/ocr3/aggregators/reduce_aggregator.go

```diff
@@ -185,6 +180,29 @@ func (a *reduceAggregator) Aggregate(lggr logger.Logger, previousOutcome *types.
 	}, nil
 }
 
+func (a *reduceAggregator) shouldReport(lggr logger.Logger, field AggregationField, singleValue values.Value, currentState *map[string]values.Value) (bool, error) {
+	if field.DeviationType == DEVIATION_TYPE_NONE {
+		return true, nil
+	}
+
+	oldValue := (*currentState)[field.OutputKey]
+	if oldValue == nil {
```

Comment:
Can we add a comment here to say that a nil value means it hasn't been initialized and this is the first pass?